crates/api_common/src/post.rs

```diff
@@ -75,6 +75,7 @@ pub struct GetPosts {
   pub community_id: Option<CommunityId>,
   pub community_name: Option<String>,
   pub saved_only: Option<bool>,
+  pub moderator_view: Option<bool>,
```

Comment:
name this `moderated_only`

Reply:
This was renamed to moderator view from moderated only because the option now implies more than just moderated only.

- moderator view also disables filtering posts from blocked users
- moderated only I think implies that only posts from a community that has moderators and enforced rules are shown, doesn't it?

Reply:
I spose that's fine.